dm/pkg/binlog/event/util.go

```diff
@@ -354,6 +354,22 @@ func GetParserForStatusVars(statusVars []byte) (*parser.Parser, error) {
 	return parser2, err
 }
 
+// GetServerCollationByStatusVars gets server collation by binlog statusVars.
+func GetServerCollationByStatusVars(statusVars []byte) (string, error) {
+	vars, err := statusVarsToKV(statusVars)
+	b, ok := vars[QCharsetCode]
+
+	if !ok {
+		if err == nil {
+			// only happen when this is a dummy event generated by DM
+			err = fmt.Errorf("Q_CHARSET_CODE not found in status_vars %v", statusVars)
+		}
+		return mysql.Collations[45], err
```

Comment:
should be "latin1_swedish_ci", which is [8] :thinking: 

https://dev.mysql.com/doc/refman/5.7/en/charset.html